compose.yml

```diff
@@ -6,30 +6,36 @@ services:
       - postgres_data:/var/lib/postgresql/data
       - ./db/postgres:/docker-entrypoint-initdb.d
     environment:
-      POSTGRES_HOST_AUTH_METHOD: trust
+      POSTGRES_USER: ats
+      POSTGRES_PASSWORD: password
+      POSTGRES_DB: ats_docker_production
     ports:
       - "5432:5432"
 
   web:
     build:
       context: .
-      dockerfile: development.Dockerfile
+      dockerfile: Dockerfile
       args:
-        - NODE_ENV=development
+        NODE_ENV: production
+        RAILS_ENV: docker_production
     command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -b '0.0.0.0'"
     volumes:
       - .:/rails
+      - /rails/public/assets # To prevent deleting precompiled assets.
```

Comment:
Looks like a HACK
Why were the assets initially deleted after precompile?

Reply:
no idea, did not look deeper

Reply:
it is exactly a hack because I think I've spend too much time on this

Reply:
seems this solution is fine because we found [another](https://github.com/nickjj/docker-rails-example/blob/61f9d7855ad52b9c23d44615112194c6460c3ec6/compose.yaml#L22-L23) similar solution